src/main.js

```diff
@@ -50,11 +55,27 @@ const renderEvents = (eventsListNode, event) => {
     eventsListNode.replaceChild(contentComponent.getNode(), editEventComponent.getNode());
   };
 
+  const onEscKeyDawn = (evt) => {
+    if (evt.key === 'Escape' || evt.key === 'Esc') {
```

Comment:
```suggestion
    if (evt.key === KeyboardKey.ESCAPE) {
```
создать енам для этого